app/setup/styleguide.styles.js

```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
+        color: '#2c68a0',
+        width: '100%',
+        textAlign: 'center'
+      },
+      heading2: {
+        'backgroundColor': '#2c68a0',
+        'color': '#fff',
+        'width': 'calc(100% + 32px)',
+        'padding': '0px 15px',
```

Comment:
- [ ] Can you align the title with the main content? Currently its a bit more to the left.